Documentation/Contribution/Index.rst

```diff
@@ -0,0 +1,30 @@
+.. include:: /Includes.rst.txt
+
+============
+Contribution
+============
+
+Unlike most official TYPO3 documentation, this manual does not support the
+popular *Edit on GitHub* workflow for manual contributions, as these
+documentation files are automatically generated from the ViewHelper
+source files - and overwritten on each generation run.
+
+Therefore, contribute to this documentation by editing the appropriate source
+files at
+
+1. https://github.com/TYPO3/typo3/tree/main/typo3/sysext/core/Classes/ViewHelpers
+2. https://github.com/TYPO3/typo3/tree/main/typo3/sysext/backend/Classes/ViewHelpers
+3. https://github.com/TYPO3/typo3/tree/main/typo3/sysext/fluid/Classes/ViewHelpers
+4. https://github.com/TYPO3/Fluid/tree/main/src/ViewHelpers
```

Comment:
Lists like these must be maintained so if at all I'd only mention the 2 repository URLs.

Reply:
This was aligned with the former CONTRIBUTING.rst and to me it makes sense, as the ViewHelper source files are deeply nested in the repositories, so i preferred deep links over simple repository links.